BranchMetrics/android-branch-deep-linking-attribution
# [SDK-2053] Removed Facebook App Links, BranchView, and v1/events code

## Reference
SDK-2053 -- Check for and remove various unnecessary code in the Android SDK

## Description
Removed old code related to Facebook App Links, BranchView, and v1/events (userCompletedAction). There was one check i am unsure about removing on line 1994 of the new `BranchUniversalObject.java` class.

## Testing Instructions
Run tests, integrate the SDK, and ensure nothing was broken or left behind in the change. 

## Risk Assessment [`MEDIUM`]


Since a lot of code was removed, there could be some breaking changes for clients.


## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style

cc @BranchMetrics/saas-sdk-devs for visibility.
